db/migrate/20150718115800_rename_spree_sales_calculator.rb

```diff
@@ -0,0 +1,9 @@
+class RenameSpreeSalesCalculator < ActiveRecord::Migration
+  def up
+    execute "UPDATE `spree_calculators` SET type = 'Spree::Calculator::AmountSalePriceCalculator' WHERE type = 'Spree::Calculator::DollarAmountSalePriceCalculator'"
```

Comment:
Note: using execute, to execute raw SQL. Alternative would be to define `Spree::Calculator::AmountSalePriceCalculator` and `Spree::Calculator::DollarAmountSalePriceCalculator` in this migration. A lot of overhead to update a set of columns to a known value.